server.js

```diff
@@ -96,8 +96,6 @@ async function startServer() {
 
 startServer();
```

Comment:
The 'app' variable is not defined. Please ensure it is defined before calling the 'route' decorator.